frontend/src/metabase/visualizer/components/DataImporter/ColumnsList/ColumnsList.tsx

```diff
@@ -92,6 +92,10 @@ export const ColumnsList = () => {
             {isExpanded && dataset && dataset.data.cols && (
               <Box ml={12} mt={2}>
                 {dataset.data.cols.map((column) => {
+                  if (column.expression_name === "pivot-grouping") {
+                    return null;
+                  }
```

Comment:
nit: could we use the [`isPivotGroupColumn` fn from `metabase/lib/data_grid`](https://github.com/metabase/metabase/blob/5242b6d15ebdd7d9ace3c3fe8bfd6a13e322d860/frontend/src/metabase/lib/data_grid.js#L8)? Just so we have less places to update if we change something about that column